samples/matter/common/src/bridge/bridge_storage_manager.cpp

```diff
@@ -160,6 +166,21 @@ bool BridgeStorageManager::RemoveBridgedDeviceEndpointId(uint8_t bridgedDeviceIn
 	return Nrf::GetPersistentStorage().NonSecureRemove(&id);
 }
 
+bool BridgeStorageManager::LoadBridgedDeviceUniqueID(char *uniqueID, size_t uniqueIDMaxLength, size_t &uniqueIDLength,
```

Comment:
I'm wondering whether this two methods: `LoadBridgedDeviceUniqueID` and `RemoveBridgedDeviceUniqueID` are needed at all. The separate load/remove methods for every attribute were part an old schema that was already replaced by the new way of loading whole device data in a single method. We left them only for the backward compatibility reasons, so they could be used to migrate older style data to the new style. To sum up I would not add them as they haven't existed in the old data structure, so they don't need to be migrated as well.

Reply:
Yes, we should remove these methods and generate UniqueID for each bridged device during the migration.

Reply:
This will be done in a follow-up task.